### ISSUE TITLE
[Release]: <v0.2.4>

### ISSUE THREAD

**Issue by BFalquet** [insightsengineering/chevron#679]
### PRs

- [x] https://github.com/insightsengineering/chevron/pull/677

### Pre-requisites

- [x] Make sure that high priority bugs (label "priority" + "bug") have been resolved before going into the release.
- [x] Review old/hanging PRs before going into the release.
- [x] Revisit R-package's lifecycle badges (Optional).
- [x] Discuss package dependencies before going into release activities.
- [x] Create a plan to sequentially close release activities and submit groups of packages for internal validation (Applicable only for regulatory release).
- [x] Make sure all relevant integration tests are green 2-3 days before the release. Look carefully through logs (check for warnings and notes).
- [x] Check if a package is installable on our supported internal systems (Optional).
- [x] Inform about the soft code freeze, decide what gets merged in before starting release activities.

### Release Checklist

- [x] Recurring tasks: Execute the manual tests on Shiny apps that are deployed on various hosting providers (Posit connect and shinyapps.io) - track the results in GitHub issue (Applicable only for frameworks that use Shiny).
- [x] Recurring tasks: Monitor integration tests, if integration fails, create priority issues on the board.
- [x] Sanity checks for Shiny applications e.g. checking if Shiny apps are deployable and making sure there are no errors/warnings.
- [x] Update NEWS.md file: make sure it reflects a holistic summary of what has changed in the package, check README.
- [x] Remove the additional fields (`Remotes` and `Config/Needs/*`) from the DESCRIPTION file where applicable.
- [x] Make sure that the minimum dependency versions are updated in the DESCRIPTION file for the package.
- [x] Increase versioned dependency on {package name} to >=X.X.X.
  - [x] dunlin 0.1.6
  - [x] formatters 0.5.4
  - [x] rlistings 0.2.6
  - [x] rtables 0.6.5
  - [x] tern 0.9.2
- [x] Create a pull request to make necessary bug fixes/changes (add "[skip vbump]" in the PR title), and after merging the PR, tag the update(s) as a release candidate v < intended release version > -rc < release candidate iteration > on the main branch. Note that tags are created in GitHub and synchronized with GitLab automatically.
- [x] The package is submitted for internal validation by Release Coordinator (Applicable only for regulatory release).
- [x] Address any feedback (internal validation/user testing), retag the package as a release candidate v0.2.4-rc(n+1). Repeat the submission for internal validation if necessary.
- [x] Get the package validated (Applicable only for regulatory release).
- [x] If the additional fields were removed, add them back in a separate PR, and then merge the PR back to main (add "[skip vbump]" in the PR title).
- [x] Create a git tag with the final version set to v0.2.4 on the main branch.
- [x] Update downstream package dependencies to (>=0.2.4) in {package name}.

### Testing

- [x] Integration tests results - accepted.
- [x] UAT results - accepted.
- [x] Shiny apps test results - accepted (Applicable only for Shiny apps).
- [x] Necessary testing on target environment - performed (up to ETL).

### Release Feedback

- [x] Fix 1
- [x] Enhancement 1
- [x] Defect 1

### Post-release Checklist

- [x] Make sure that the package is published to internal repositories (Validated and/or Non-Validated repository).
- [x] Review and update installation instructions for the package if needed.
- [x] Verify if a new dev version (.9XXX) has been added to the NEWS.md file and DESCRIPTION file as a placeholder for release notes by automation.
- [x] Make sure internal documentation/documentation catalogs are up to date.
- [x] Notify the IDR team to start post-release/clean-up activities.
- [x] Announce the release on ________.

### Decision tree

Click [here](https://github.com/insightsengineering/.github/blob/main/.github/ISSUE_TEMPLATE/RELEASE_DECISION_TREE.md) to see the release decision tree.
